legal-api/src/legal_api/utils/legislation_datetime.py

```diff
@@ -37,12 +37,17 @@ def tomorrow_midnight():
         return date
 
     @staticmethod
-    def as_legislation_timezone(date_time: datetime):
+    def as_legislation_timezone(date_time: datetime) -> datetime:
         """Return a datetime adjusted to the legislation timezone."""
         return date_time.astimezone(pytz.timezone(current_app.config.get('LEGISLATIVE_TIMEZONE')))
 
     @staticmethod
-    def format_as_legislation_date(date_string: str):
+    def as_utc_timezone(date_time: datetime) -> datetime:
+        """Return a datetime adjusted to the GMT timezone (aka UTC)."""
+        return date_time.astimezone(pytz.timezone('GMT'))
+
+    @staticmethod
+    def format_as_legislation_date(date_string: str) -> str:
```

Comment:
doesn't this return a -> datetime ?

Reply:
No, the chained return statement below returns a formatted string (strftime).